src/sentry/static/sentry/app/views/settings/account/avatar.jsx

```diff
@@ -101,29 +105,61 @@ const AvatarSettings = React.createClass({
     );
 
     return (
-      <div style={{lineHeight: '1.5em'}}>
-        <form>
-          <AvatarRadio user={this.state.user} updateUser={this.updateUserState} />
-
-          {this.state.user.avatar.avatarType === 'gravatar' && gravatarMessage}
-
-          {this.state.user.avatar.avatarType === 'upload' && (
-            <AvatarCropper
-              {...this.props}
-              user={this.state.user}
-              savedDataUrl={this.state.savedDataUrl}
-              updateDataUrlState={this.updateDataUrlState}
+      <Panel>
+        <PanelHeader>Avatar</PanelHeader>
+        <PanelBody>
+          <AvatarForm>
+            <RadioGroup
+              choices={[
+                ['letter_avatar', 'Use my initials'],
+                ['upload', 'Upload a Photo'],
+                ['gravatar', 'Use Gravatar'],
+              ]}
+              value={this.state.user.avatar.avatarType || 'letter_avatar'}
+              label="Avatar Type"
+              onChange={(id, e) =>
```

Comment:
There's a bit going on here, extract it into a separate function.

Instead of `Object.assign` I'd suggest:
`onChange={id => handleChange(id)}`

```
handleChange(id) {
    let user = {...this.state.user};
    user.avatar.avatarType = id;
    this.updateUserState(user);
}
```

Reply:
@lynnagara Totally agree. I wanted to come back and make this code better, but then suffered from a bout of developer amnesia and never got to it. Thank you got catching this!